Automattic/wp-calypso
# Remove migration title to match the design

## Proposed Changes

* It just removes the flow title.

## Why are these changes being made?


* To match the design.

## Testing Instructions



* Navigate to `/setup/migration`, and make sure it doesn't have the title in the side of the WordPress logo anymore.

#### Before:

<img width="499" alt="Screenshot 2024-08-14 at 12 17 00" src="https://github.com/user-attachments/assets/0f1de791-6781-482e-bb23-4676854546e9">

#### After:

<img width="487" alt="Screenshot 2024-08-14 at 12 15 54" src="https://github.com/user-attachments/assets/e959abd0-ddea-4dd4-8973-4002a1fcca61">


## Pre-merge Checklist
